cumulusci.yml

```diff
@@ -43,6 +43,13 @@ tasks:
                 Id guestId = [SELECT ID From User WHERE Name = 'Volunteers Site Guest User' LIMIT 1][0].id;
                 insert new PermissionSetAssignment(PermissionSetId=psetId, AssigneeId=guestId);
 
+    test_data:
```

Comment:
@davidmreed - I added a task and a script to generate test data for this bug. Github shows release engineering as the code owner and suggested it be reviewed by a member of the RE team. Thanks!

Reply:
Looks good to me, @jjbennett!

Reply:
@davidmreed - yes it's just for easier regression test at this point, the idea is to maybe also use it by a robot test in the future.